### PR TITLE
Restructure and Expand Meeting Add-in Troubleshooting

### PR DESCRIPTION
**Pull request by chtartMSFT**
Adding and clarifying guidance for Troubleshooting Windows add-in issues.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Teams-add-in-for-Outlook.md [hunk 1]
```diff
@@ -109,19 +109,50 @@ The Teams Meeting add-in is still building functionality, so be aware of the fol
...
+## Teams Meeting add-in in Outlook for Windows does not show
 
 If you cannot get the Teams Meeting add-in for Outlook to install, try these troubleshooting steps.
 
-- Ensure all available updates for Outlook desktop client have been applied.
-- Restart the Teams desktop client.
-- Sign out and then sign back in to the Teams desktop client.
-- Restart the Outlook desktop client. (Make sure Outlook isn't running in admin mode.)
+- Check that the user has a Teams Upgrade policy which enables scheduling meetings in Teams. See [Upgrade from Skype for Business to Teams](https://docs.microsoft.com/microsoftteams/upgrade-to-teams-on-prem-overview#meetings) for more details.
+- Check that the user has a Teams Meeting policy which permits the Outlook Add-in. See [Manage meeting policies in Teams](https://docs.microsoft.com/microsoftteams/meeting-policies-in-teams#allow-the-outlook-add-in) for more details.
```
Comment (JohanFreelancer9) [inline Teams/Teams-add-in-for-Outlook.md:123]:
```suggestion
- Check that the user has a Teams Meeting policy that permits the Outlook Add-in. See [Manage meeting policies in Teams](https://docs.microsoft.com/microsoftteams/meeting-policies-in-teams#allow-the-outlook-add-in) for more details.
```

File: Teams/Teams-add-in-for-Outlook.md [hunk 2]
```diff
@@ -109,19 +109,50 @@ The Teams Meeting add-in is still building functionality, so be aware of the fol
...
+- Ensure the user has the Teams desktop client installed.  The meeting add-in will not be installed when only using the Teams web client.
```
Comment (JohanFreelancer9) [inline Teams/Teams-add-in-for-Outlook.md:124]:
```suggestion
- Ensure that the user has the Teams desktop client installed. The meeting add-in will not be installed when only using the Teams web client.
```

File: Teams/Teams-add-in-for-Outlook.md [hunk 3]
```diff
@@ -109,19 +109,50 @@ The Teams Meeting add-in is still building functionality, so be aware of the fol
...
+- Make sure the user has permission to execute regsvr32.exe.
```
Comment (JohanFreelancer9) [inline Teams/Teams-add-in-for-Outlook.md:125]:
```suggestion
- Make sure that the user has permission to execute regsvr32.exe.
```

File: Teams/Teams-add-in-for-Outlook.md [hunk 4]
```diff
@@ -109,19 +109,50 @@ The Teams Meeting add-in is still building functionality, so be aware of the fol
...
+- Ensure the user is running the click-to-run version of Office 2016 or later and that all available updates for Outlook desktop client have been applied.
```
Comment (JohanFreelancer9) [inline Teams/Teams-add-in-for-Outlook.md:126]:
```suggestion
- Ensure that the user is running the click-to-run version of Office 2016 or later and that all available updates for Outlook desktop client have been applied.
```

File: Teams/Teams-add-in-for-Outlook.md [hunk 5]
```diff
@@ -109,19 +109,50 @@ The Teams Meeting add-in is still building functionality, so be aware of the fol
...
 - Make sure the logged-in user account name does not contain spaces. (This is a known issue, and will be fixed in a future update.)
-- Make sure single sign-on (SSO) is enabled.
+- Follow these steps:
+  - Restart the Teams desktop client.
+  - Sign out and then sign back in to the Teams desktop client.
+  - Restart the Outlook desktop client. (Make sure Outlook isn't running in admin mode)
```
Comment (JohanFreelancer9) [inline Teams/Teams-add-in-for-Outlook.md:131]:
```suggestion
  - Restart the Outlook desktop client. (Make sure Outlook isn't running in admin mode.)
```

File: Teams/Teams-add-in-for-Outlook.md [hunk 6]
```diff
@@ -109,19 +109,50 @@ The Teams Meeting add-in is still building functionality, so be aware of the fol
...
+
+If you still don't see the add-in, make sure that it isn't disabled in Outlook.
+
+- In Outlook, click **File** and then **Options**.
```
Comment (JohanFreelancer9) [inline Teams/Teams-add-in-for-Outlook.md:135]:
```suggestion
- In Outlook, choose **File** and then **Options**.
```

File: Teams/Teams-add-in-for-Outlook.md [hunk 7]
```diff
@@ -109,19 +109,50 @@ The Teams Meeting add-in is still building functionality, so be aware of the fol
...
+- Select the **Add-ins** tab of **Outlook Options** dialog box
```
Comment (JohanFreelancer9) [inline Teams/Teams-add-in-for-Outlook.md:136]:
```suggestion
- Select the **Add-ins** tab of **Outlook Options** dialog box.
```

File: Teams/Teams-add-in-for-Outlook.md [hunk 8]
```diff
@@ -109,19 +109,50 @@ The Teams Meeting add-in is still building functionality, so be aware of the fol
...
+- Confirm that **Microsoft Teams Meeting Add-in for Microsoft Office** is listed in the **Active Application Add-ins** list
+- If the Teams Meeting Add-in is listed in the **Disabled Application Add-ins** list, select **COM Add-ins** in **Manage** and click **Go…**
```
Comment (JohanFreelancer9) [inline Teams/Teams-add-in-for-Outlook.md:138]:
```suggestion
- If the Teams Meeting Add-in is listed in the **Disabled Application Add-ins** list, select **COM Add-ins** in **Manage** and select **Go…**
```

File: Teams/Teams-add-in-for-Outlook.md [hunk 9]
```diff
@@ -109,19 +109,50 @@ The Teams Meeting add-in is still building functionality, so be aware of the fol
...
+- Set the checkbox next to **Microsoft Teams Meeting Add-in for Microsoft Office**
```
Comment (JohanFreelancer9) [inline Teams/Teams-add-in-for-Outlook.md:139]:
```suggestion
- Set the checkbox next to **Microsoft Teams Meeting Add-in for Microsoft Office**.
```

File: Teams/Teams-add-in-for-Outlook.md [hunk 10]
```diff
@@ -109,19 +109,50 @@ The Teams Meeting add-in is still building functionality, so be aware of the fol
...
+- Click OK on all dialog boxes and restart Outlook.
```
Comment (JohanFreelancer9) [inline Teams/Teams-add-in-for-Outlook.md:140]:
```suggestion
- Choose OK on all dialog boxes and restart Outlook.
```

File: Teams/Teams-add-in-for-Outlook.md [hunk 11]
```diff
@@ -109,19 +109,50 @@ The Teams Meeting add-in is still building functionality, so be aware of the fol
 - The add-in will not work if an Authentication Proxy is in the network path of user's PC and Teams Services.
```
Comment (JohanFreelancer9) [inline Teams/Teams-add-in-for-Outlook.md:109]:
```suggestion
- The add-in will not work if an Authentication Proxy is in the network path of the user's PC and Teams Services.
```

**Comment by scanum** [comment]
@chuckedmonson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**